pkgs/misc/mox-boot-builder/default.nix

```
@@ -0,0 +1,46 @@
+{ stdenv
+, lib
+, fetchgit
+, buildPackages
+, pkgsCross
+}:
+
+stdenv.mkDerivation rec {
+  pname = "mox-boot-builder";
+  version = "v2022.08.30";
+  src = fetchgit {
+    url = "https://gitlab.nic.cz/turris/mox-boot-builder.git";
+    rev = "0290b2cd9e14041ef7bcd267840f30d6c0a92ceb";
+    hash = "sha256-ZlbgJj/KgiDcF7Jl+wdNJi7iSivBIGF0W1qbrcIWBvk=";
+    fetchSubmodules = false;
+  };
+
+  installDir = "$out";
+
+  nativeBuildInputs = [ pkgsCross.arm-embedded.stdenv.cc ];
+
+  makeFlags = [
+    "CROSS_CM3=${buildPackages.gcc-arm-embedded}/bin/arm-none-eabi-"
+    "wtmi_app.bin"
+  ];
+
+  filesToInstall = ["wtmi_app.bin"];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p ${installDir}
+    cp ${lib.concatStringsSep " " filesToInstall} ${installDir}
```

```suggestion
    mkdir -p $out
    cp wtmi_app.bin $out
```

We don't neeed the level of indirection here. `$out` is the proper way to refer to the (default) output. We don't have enough complexity to warrant concating this way.

> **TIP** in this case `lib.escapeShellArgs` would have been preferred, as it would correctly handle complex cases.